lib/pure/json.nim

```diff
@@ -908,15 +908,14 @@ proc parseJson*(s: Stream, filename: string = ""; rawIntegers = false, rawFloats
 
 when defined(js):
   from std/math import `mod`
-  type
-    JSObject = object
+  import std/jsffi
 
-  proc parseNativeJson(x: cstring): JSObject {.importc: "JSON.parse".}
+  proc parseNativeJson(x: cstring): JSObject {.importjs: "JSON.parse(#)".}
 
   proc getVarType(x: JSObject): JsonNodeKind =
     result = JNull
     proc getProtoName(y: JSObject): cstring
```

Comment:
how about moving this to `std/private/jsutils`, it fits well along with the other `jsTypeOf`, `jsConstructorName` etc. And the proc could be changed to:
```nim
proc getProtoName*[T](a: T): cstring
```

Reply:
make sense